pkg/asset/installconfig/gcp/gcp.go

```diff
@@ -38,16 +38,48 @@ func selectProject(ctx context.Context) (string, error) {
 	}
 	defaultProject := ssn.Credentials.ProjectID
 
+	client := &Client{
+		ssn: ssn,
+	}
+
+	var listOfCombo []string
```

Comment:
A better name would be something like `projectChoices`. Don't begin variable names with types or pseudo types like ` list`; `combo` is vague.